### DISCUSSION TITLE
Best method to label cells between 2 biological samples

### DISCUSSION THREAD

**Discussion by huberax** [satijalab/seurat#7224] [Q&A]
Hello all,

I am fairly new to single cell RNAseq analysis and Seurat and wanted to ask a question about properly naming/annotating data between 2 sets of biological samples. 

For this question, let's say we have one cell type enriched from different tissue compartments at homeostasis and during infection.

From the papers I have read, I have seen 2 methods of analyzing and annotating subpopulations of a dataset between the uninfected and infected group:

**i**) Cluster, characterize, and label the cells of the control group first (to understand what cell types are present at homeostasis first),  and then integrate the control and infected group together and transfer your new labels to the infected group (and exclude cells with low prediction scores). Then, compare and contrast differences between uninfected and infected host clusters.

**ii**) Start by integrating the data together, and then characterizing and labeling the clusters. Afterwards, compare and contrast differences between uninfected and infected host clusters.

The things I have noticed when testing these methods out are:

**a**) Cells of an uninfected host alone will cluster slightly different than the infected host alone, yielding different seurat clusters (with a specific set of DEGs) that may not necessarily be found in the integrated data (While the cells are still in the integrated dataset, they have clustered differently). 

**b**) When I transfer the labels over, it seems like I'm assuming that all of my cell types/subpopulations from the uninfected host will be in the infected host. 

**c**) For label transfer methods, it is my understanding that you exclude cells with low prediction scores, where you cannot confidently annotate the cell based off genes expressed in the reference cluster. However, you do lose cells that are the cell type of interest (i.e. monocyte), but do not align with the specific cluster of the reference ("cluster3 monocytes" for example).  

I am more prone to perform the second approach in which I find the pair-wise anchors between the datasets first, and perform analysis that way. 

Does anyone have any advice that can help steer me one way or another? And again, I am fairly new to this stuff, so please let me know if you need me to clarify any points.

**Comment by Jiayi-Zheng** [comment]
I usually do it separately first, if it's not too much extra work. Then do integration and do manual label transfer with caution (that's what I did for mine, so I don't spend too much time on labeling cells from each dataset). 
Batch effect correction is a headache.

**Reply by huberax** [reply]
Hello @Jiayi-Zheng ,

Thank you for your response. I was more curious in the concept and what is the best way to approach the issue and why. Would you be able to expand on those points?

Thank you

**Reply by Jiayi-Zheng** [reply]
I'm also new to seq data analysis so just take what I said as reference only...
One of the major benefit of merging multiple datasets is the detection of rare cell type -- with multiple datasets they are able to form small cluster and become detectable. However I don't feel like this is an expectation for investigating two different conditions. On the other hand, batch correction is finding balance between ignoring meaningful biological features and eliminating technical noises. No matter how you choose, it always cause certain effect on the biological features of the cells. So I feel like it should be used with caution, hence I personally prefer to do analysis on separate dataset first.
